pkg/logql/count_min_sketch_test.go

```diff
@@ -68,7 +71,7 @@ func TestCountMinSketchSerialization(t *testing.T) {
 		Sketch: &logproto.CountMinSketch{
 			Depth:       2,
 			Width:       4,
-			Counters:    []float64{0, 0, 0, 42, 0, 42, 0, 0},
+			Counters:    []float64{0, 42, 0, 0, 0, 42, 0, 0},
```

Comment:
why was this change needed? how was the test failing?

Reply:
Using the `labels.Bytes` change that hash values. That's why the position of 42 is different.